### DISCUSSION TITLE
Proxy Manager with Nginx on system?

### DISCUSSION THREAD

**Discussion by gltched-usr** [NginxProxyManager/nginx-proxy-manager#2744] [Q&A]
I have already nginx installed on my server(default installation, without docker). 
But Proxymanager uses the same ports I use in nginx(80 and 443 for https).

Can I use the already installed nginx or if not, is there an easy way to migrate my current nginx config to nginx proxy manager(or at least nginx docker if that works).

**Comment by cosmicchaos80** [comment]
It's been a while, but I have the same question. Nextcloud running with nginx. Now I want to switch from nginx-proxy-manager to Docker. conf.d and letsencrypt are copied to the container. Unfortunately the Nextcloud configuration is not shown in nginx-progy-manager. Access to the Nextcloud is therefore no longer possible.
The old nginx configuration has been copied to the folder /path/to/container/etc/nginx/conf.d. But as I said, unfortunately it cannot be found in the nginx-proxy-manager interface. Ports are of course already redirected to nginx-proxy-manager and nginx is disabled. Any ideas?

**Comment by luisedo97** [comment]
Currently, I'm developing a Node.js code to extract all the hosts in my current Nginx installation. I plan to stop the Nginx service, activate NPM, and run the command to leverage the NPM API to load the new hosts one by one. In theory, and based on a couple of tests I've performed on another VPS, everything works fine. However, there are around 500 domains I need to migrate, so let's hope everything goes well. The worst-case scenario would be having to stop the migration, halt NPM, and reactivate the Nginx service.
